Source/Core/VideoCommon/TextureCacheBase.cpp

```diff
@@ -756,15 +756,15 @@ TextureCacheBase::TCacheEntryBase* TextureCacheBase::Load(const u32 stage)
   }
 
   // how many levels the allocated texture shall have
-  const u32 texLevels = hires_tex ? (u32)hires_tex->m_levels.size() : tex_levels;
+  const u32 texLevels = custom_tex ? (u32)custom_tex->m_levels.size() : tex_levels;
```

Comment:
Unfortunately I don't understand the terms in the advice well enough to figure out what I'm suppose to do.